Documentation/configuring-storage.md

```diff
@@ -56,20 +56,64 @@ The `spec.storage.hive.s3Compatible.secretName` should be the name of a secret i
 
 > **Note**: The values of the `aws-access-key-id` and `aws-secret-access-key` must be base64 encoded.
 
-For example:
-
+Noobaa installation example:
+1. get noobaa CLI
+     * download [noobaa CLI tool](https://github.com/noobaa/noobaa-core) and put it into your bin PATH
+2. install noobaa in your cluster
+```bash
+oc create ns noobaa
```

Comment:
Do you all typically leave the `$` out of `oc` commands? If so, please ignore this comment.

Reply:
I believe that's the markdown recommendation is to not prefix a bash command with `$` w/o command output that proceeds that command.